b9/include/b9/ExecutionContext.hpp

```diff
@@ -8,17 +8,46 @@
 
 namespace b9 {
 
+// Stack layout in Base9:
+//
+// |---------- <<< stack upper limit.
+// | ...
+// |---------- <<< top pointer
+// | Target Scratch Space:
+// | * operand 0
+// | * operand N
+// |---------- <<< base pointer
+// | Stack Frame (fixed size):
+// | * caller base pointer
+// | * caller instruction pointer (could be an offset into function's
```

Comment:
messed up text relfow

Reply:
damn! will fix. Also, got the arg0-argN and var0 - varN order wrong.

Reply:
also maybe some kind of clearer notation that, when reading from top to bottom, addresses are high to low